marklogic-data-hub-central/ui/e2e/cypress/integration/explore/exploreViewSelector.tsx

```diff
@@ -0,0 +1,93 @@
+import {Application} from "../../support/application.config";
+import browsePage from "../../support/pages/browse";
+import {toolbar} from "../../support/components/common";
+import LoginPage from "../../support/pages/login";
+
+/**
+ * NOTE: This test will involve all operations related to the views selector.
+ */
+
+describe("Test '/Explore' view selector", () => {
+  before(() => {
+    cy.visit("/");
+    cy.contains(Application.title);
+
+    cy.log("**Logging into the app as a developer**");
+    cy.loginAsDeveloper().withRequest();
+    LoginPage.postLogin();
+    //Saving Local Storage to preserve session
+    cy.saveLocalStorage();
+  });
+  beforeEach(() => {
+    //Restoring Local Storage to Preserve Session
+    Cypress.Cookies.preserveOnce("HubCentralSession");
+    cy.restoreLocalStorage();
+  });
+  it(`Validate that the 'graph' view is shown and stored in the user preference`, () => {
+    cy.log(`**Go to Explore section?**`);
+    toolbar.getExploreToolbarIcon().click();
+
+    cy.log(`**Selecting 'Graph' view**`);
+    browsePage.viewSelector("graph").should("exist").click();
+    browsePage.getGraphVisExploreContainer().should("exist");
+
+    cy.log(`**Go to Monitor section**`);
+    toolbar.getMonitorToolbarIcon().click();
```

Comment:
You can add a simple validation here simply to confirm that it actually went to monitor section. For example a title